web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -69,20 +80,45 @@ export class LayerDialogComponent implements OnDestroy {
     this.projectId = project.id;
   }
 
+  getForm(question: string, fieldId: string, formId: string): Form {
```

Comment:
I assume this function is a temporary function until we replace with real form designer functionality? If so no action needed (yet).

Reply:
Yes this is just a temporary function for now :)

Reply:
Sgtm